aten/src/ATen/core/function_schema.h

```diff
@@ -146,12 +155,17 @@ struct FunctionSchema {
     return is_varret_;
   }
   bool is_mutable() const {
-    return std::any_of(
-        arguments_.cbegin(), arguments_.cend(), [](const Argument& arg) {
-          const auto& aliasInfo = arg.alias_info();
-          return aliasInfo && aliasInfo.value().isWrite();
-        });
+    return is_custom_op_ ||
```

Comment:
Nit: is the intention that the custom operator returns true even if there are no arguments ? if not, than the is_custom_op_ isn't needed. Otherwise maybe add a comment?